.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 repos:
   - repo: git://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.9
+    rev: v1.1.13
```

Comment:
Please take the opportunity to switch from `git://` to `https://`  (ref https://github.com/thoth-station/thoth-application/issues/2111)